src/core/OmniInputStories.ts

```diff
@@ -264,6 +264,53 @@ const App = () => <${toPascalCase(tagName)}${args.label ? ` label='${args.label}
     return Clearable;
 };
 
+export const CustomClearableSlotIcon = <T extends HTMLElement, U extends BaseArgs>(
+    tagName: string,
+    inputValue: string | number | string[] = 'The input value'
+) => {
+    const Clearable: ComponentStoryFormat<U> = {
+        render: (args: U) =>
+            html`${unsafeHTML(
+                `<${tagName} data-testid="test-field" label="${ifNotEmpty(args.label)}" value="${args.value}" clearable>
+                ${args.clear}
+                </${tagName}>`
+            )}`,
+        frameworkSources: [
+            {
+                framework: 'React',
+                load: (args) => `import { ${toPascalCase(tagName)} } from "@capitec/omni-components-react/${asDirectoryName(tagName)}";
+    
+                const App = () => <${toPascalCase(tagName)}${args.label ? ` label='${args.label}'` : ''}${
+                    args.value ? ` value='${args.value}'` : ''
+                } clearable>
+                    <svg slot="clear" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24px" height="24px" style="fill: orange; margin-left: 10px;"><path d="M12 2.25c5.385 0 9.75 4.365 9.75 9.75s-4.365 9.75-9.75 9.75S2.25 17.385 2.25 12 6.615 2.25 12 2.25Zm0 1.5a8.25 8.25 0 1 0 0 16.5 8.25 8.25 0 0 0 0-16.5ZM12 7a.75.75 0 0 1 .75.75v3.5h3.5a.75.75 0 0 1 .743.648L17 12a.75.75 0 0 1-.75.75h-3.5v3.5a.75.75 0 0 1-.648.743L12 17a.75.75 0 0 1-.75-.75v-3.5h-3.5a.75.75 0 0 1-.743-.648L7 12a.75.75 0 0 1 .75-.75h3.5v-3.5a.75.75 0 0 1 .648-.743Z"/></svg>
```

Comment:
Please check the React snippet. Those style attributes probably don't work

Reply:
Updated accordingly and tested in Codepen please see the attached

![image](https://github.com/capitec/omni-components/assets/22874454/bee19cee-8647-4c32-9369-6086fda18753)